Panic when running on Windows 8

Running vault in Windows 8 immediately panics with both the 32 and 64-bit prebuilt binaries.

```
PS C:\Users\William\Downloads> .\vault.exe
panic: Failed to find SetConsoleMod procedure in kernel32.dll: The specified procedure could not be found.

goroutine 1 [running]:
syscall.(*DLL).MustFindProc(0xc082006bc0, 0xa6c830, 0xd, 0xc082008b70)
        /Users/mitchellh/code/3rdparty/go/src/syscall/dll_windows.go:95 +0x88
github.com/hashicorp/vault/helper/password.init()
        /Users/mitchellh/code/go/src/github.com/hashicorp/vault/helper/password/password_windows.go:12 +0xff
github.com/hashicorp/vault/command.init()
        /Users/mitchellh/code/go/src/github.com/hashicorp/vault/command/write.go:122 +0xdf
github.com/hashicorp/vault/cli.init()
        /Users/mitchellh/code/go/src/github.com/hashicorp/vault/cli/version.go:13 +0x67
main.init()
        /Users/mitchellh/code/go/src/github.com/hashicorp/vault/main.go:11 +0x49

goroutine 2 [runnable]:
runtime.forcegchelper()
        /Users/mitchellh/code/3rdparty/go/src/runtime/proc.go:90
runtime.goexit()
        /Users/mitchellh/code/3rdparty/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 3 [runnable]:
runtime.bgsweep()
        /Users/mitchellh/code/3rdparty/go/src/runtime/mgc0.go:82
runtime.goexit()
        /Users/mitchellh/code/3rdparty/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 4 [runnable]:
runtime.runfinq()
        /Users/mitchellh/code/3rdparty/go/src/runtime/malloc.go:712
runtime.goexit()
        /Users/mitchellh/code/3rdparty/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 5 [runnable]:
os/signal.loop()
        /Users/mitchellh/code/3rdparty/go/src/os/signal/signal_unix.go:19
created by os/signal.init┬╖1
        /Users/mitchellh/code/3rdparty/go/src/os/signal/signal_unix.go:27 +0x3c
```

--- a/helper/password/password_windows.go
+++ b/helper/password/password_windows.go
@@ -9,7 +9,7 @@ import (
 
 var (
 	kernel32           = syscall.MustLoadDLL("kernel32.dll")
-	setConsoleModeProc = kernel32.MustFindProc("SetConsoleMod")
+	setConsoleModeProc = kernel32.MustFindProc("SetConsoleMode")
 )
 
 // Magic constant from MSDN to control whether charactesr read are

